scripts/custom-screens/CustomFormControls.tsx

```diff
@@ -25,90 +30,109 @@ const CustomFormControls = () => {
-          <DateField
-              entityName="scr_CarRent"
-              propertyName="fromDate"
-              formItemProps={{
-                style: { marginBottom: "12px" }
-              }}
-          />
-          <TimeField
-              entityName="scr_CarRent"
-              propertyName="fromTime"
-              formItemProps={{
-                style: { marginBottom: "12px" }
-              }}
-          />
-          <CheckboxField
-              entityName="scr_Car"
-              propertyName="wheelOnRight"
-              formItemProps={{
-                style: { marginBottom: "12px" }
-              }}
-          />
-          <ColorPickerField
-              entityName="scr_Car"
-              propertyName="color"
-              formItemProps={{
-                style: { marginBottom: "12px" }
-              }}
-          />
-
+    <Card className={styles.narrowLayout}>
+      <Form
+        onFinish={handleSubmit}
+        onFinishFailed={() => {}}
+        layout="vertical"
+        form={form}
+        initialValues={{
+          manufacturer: "Initial Manufacturer",
+          regNumber: "Initial Reg Number",
+          mileage: 100000,
+          price: [400000, 1500000],
+          wheelOnRight: true,
+          fromDate: dayjs("2020-01-01"),
+          fromTime: dayjs("2020-01-01T23:05:13")
+        }}
+        validateMessages={createAntdFormValidationMessages(intl)}
+      >
+        <TextField
+          entityName="scr_Car"
+          propertyName="manufacturer"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+        />
+        <DateField
+          entityName="scr_CarRent"
+          propertyName="fromDate"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+        />
+        <TimeField
+          entityName="scr_CarRent"
+          propertyName="fromTime"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+        />
+        <CheckboxField
+          entityName="scr_Car"
+          propertyName="wheelOnRight"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+        />
+        <ColorPickerField
+          entityName="scr_Car"
+          propertyName="color"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+        />
         <TextArea
           entityName="scr_Car"
           propertyName="regNumber"
           formItemProps={{
             style: { marginBottom: "12px" }
           }}
         />
+        <Slider
+          entityName="scr_Car"
+          propertyName="mileage"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+          max={300000}
+          tooltipVisible
+        />
+        <SliderRange
+          entityName="scr_Car"
+          propertyName="price"
+          formItemProps={{
+            style: { marginBottom: "12px" }
+          }}
+          range
```

Comment:
Need to remove this prop(range: boolean) from SliderRange or combine Slider and SliderRange:
```typescript
type SliderProps = (SliderSingleProps | SliderRangeProps) & JmixFormFieldProps;
```
I would like to combine interfaces because realizations are the same and this Slider will be closer to ant Slider.